### PR TITLE
Group placement

### PR DESCRIPTION
**Pull request by MikhailZvagelsky**

### CODE REVIEW

File: api/data/series/group.md [hunk 1]
```diff
@@ -560,3 +622,365 @@ The grouped `SUM` series is then aggregated into periods.
...
+
+* For each subgroup calculate value of the `minimize` expression, which can use the same [statistical functions](#functions-available-in-the-place-context).
+
+* Calculate value of objective function as sum of values calculated on the previous step.
+
+The solution is a partition which minimizes the objective function. If objective function reaches minimum on several partitions then select one of them with the least count of subgroups.
+
+#### Partitioning Algorithm Complexity
+
+Algorithm loops over all partitions of set of cardinality N into no more then K subsets.
```
Comment (raipc) [inline api/data/series/group.md]: than

File: api/data/series/group.md [hunk 2]
```diff
@@ -560,3 +622,365 @@ The grouped `SUM` series is then aggregated into periods.
...
+Total count of partitions equals to sum S(N, 1) + S(N, 2) + ... + S(N, K)
+of [Stirling](https://en.wikipedia.org/wiki/Stirling_numbers_of_the_second_kind) numbers.
+This sum grows extremely fast, for example, there are `16_244_652_278_171` partitions of 20 series into no more then 7 subgroups.
```
Comment (raipc) [inline api/data/series/group.md]: than

File: api/data/series/group.md [hunk 3]
```diff
@@ -560,3 +622,365 @@ The grouped `SUM` series is then aggregated into periods.
...
+      {"d": "2019-02-07T22:00:00.000Z", "v": 5},
+      {"d": "2019-02-07T23:00:00.000Z", "v": 5}
+    ]
+  }
+]
+```
+
+### Packing
+
+The problem is to divide series into the smallest number of subgroups, and each subgroup meets the `constraint`.
```
Comment (raipc) [inline api/data/series/group.md]: where

File: api/data/series/group.md [hunk 4]
```diff
@@ -560,3 +622,365 @@ The grouped `SUM` series is then aggregated into periods.
...
+| lp_1              |                4 |
+| lp_2              |                4 |
+| lp_3              |                4 |
+| lp_4              |                8 |
+| lp_5              |                4 |
+| lp_6              |                2 |
+| lp_7              |                4 |
+```
+
+The following `constraint` asserts that sum of `memory_allocated` parameters of all series in a subgroup is less then 14:
```
Comment (raipc) [inline api/data/series/group.md]: than

File: api/data/series/group.md [hunk 5]
```diff
@@ -560,3 +622,365 @@ The grouped `SUM` series is then aggregated into periods.
...
+```json
+"constraint": "max() < cpu_limit && sum(series.memory_allocated) < memory_limit"
+```
+
+It asserts that maximum value of aggregated series computed for `lp_2, lp_3, lp_6` is less than `cpu_limit = 10`, and sum of memory allocated to each series `sum(series.memory_allocated) = 4 + 4 + 2` is
+less that `memory_limit = 20`.
+
+#### Packing Algorithm
+
+Exact solution of the Packing problem requires consider each of
```
Comment (raipc) [inline api/data/series/group.md]: requires to

File: api/data/series/group.md (hunk 2 above)
Comment (MikhailZvagelsky) [reply api/data/series/group.md]: Thank you! Fixed.

**Pull request merged**